File: Python/Simple-Fizzbuzz.py
#Probably the simplest and dumbest fizzbuzz you can find in this folder
#Author:Sourav Nanda
for num in range(1,101):
    if num%3==0:
        print('fizz')
    elif num%5==0:
        print('buzz')
    elif num%5==0 and num%3==0:
        print('fizzbuzz')
    else:
        print(num)

File: Rust/simpleFizzBuzz.rs
fn checker(num:isize, n:isize) -> bool {
    if num % n == 0 {
        true
    } else {
        false
    } 
}
fn main() {
let mut line = String::new()
    let mut startAt = 1;
    let max = 99;
    while start <= max {
        match (checker(startAt, 3), checker(startAt, 5)) {
            (true,true) => println!("FizzBuzz"),
            (true,false) => println!("Fizz"),
            (false,true) => println!("Buzz"),
            (false,false) => println!("{}", startAt)
        }

        startAt+=1;
    }
}
